pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -86,6 +88,18 @@ func CmdRegenerate() *cobra.Command {
 	return cmd
 }
 
+// CmdCredentials returns a CLI command
+func CmdCredentials() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "credentials <noobaa-account-name>",
+		Short: "Update S3 Credentials(access-key and secret-key)",
+		Run:   RunCredentials,
+	}
+	cmd.Flags().String("access_key", "", "Update the access_key for an account")
+	cmd.Flags().String("secret_key", "", "Update the secret_key for an account")
```

Comment:
I think you should warn the user that using the flag is not secure, for example:
https://github.com/noobaa/noobaa-operator/blob/fcc5240f5af690596823f2ab63e9debd9a640fda/pkg/namespacestore/namespacestore.go#L82

Reply:
Make sense, will update it. Thanks